src/redux/categories/CategoriesSlice.js

```diff
@@ -0,0 +1,19 @@
+// Actions
+const CHECK_STATUS = 'bookstore/categories/CHECK_STATUS';
+
+// Reducer
+export default function categoriesReducer(state = [], action) {
+  switch (action.type) {
+    case CHECK_STATUS:
+      return 'Under construction';
+
+    default:
+      return state;
+  }
+}
+
+// Action Creators
+
+const checkStatusAction = () => ({ type: CHECK_STATUS });
+
+export { CHECK_STATUS, checkStatusAction };
```

Comment:
- [ ] Good job following the duck pattern in this file and its working but the issue is that you are required in this milestone to use the new and better way to write action creates and reducers which is through redux toolkit's feature createSlice. kindly consider writing the same functionality using createSlice feature of redux toolkit. you need to implement following the requirements below and you can also take help from redux toolkit official documentation.
- Define a slice of state for categories that:
   -  Includes an array of categories (initial state: empty array)
   -  Includes a reducer that checks the status and always returns "Under construction" (the initial state should check to that string)